components/service/glean/src/main/java/mozilla/components/service/glean/private/TimespanMetricType.kt

```diff
@@ -30,39 +32,58 @@ data class TimespanMetricType(
     private val logger = Logger("glean/TimespanMetricType")
 
     /**
-     * Start tracking time for the provided metric. This records an error if it’s
-     * already tracking time (i.e. start was already called with no corresponding
-     * [stopAndSum]): in that case the original start time will be preserved.
+     * Start tracking time for the provided metric and associated object. This
+     * records an error if it’s already tracking time (i.e. start was already
+     * called with no corresponding [stopAndSum]): in that case the original
+     * start time will be preserved.
+     *
+     * @param anyObject The object to associate with this timing.  This allows
```

Comment:
nit: I would be a bit more explicit with the name of the parameter here. Maybe something like `timerId` instead of `anyObject`